docs/Project.toml

```diff
@@ -0,0 +1,9 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+MCMCChains = "c7f686f2-ff18-58e9-bc7b-31028e88f75d"
```

Comment:
It seems the MCMCChains dependency reappeared:
```suggestion
```

Reply:
My bad. Fixed again.